upgradeTestRail.sh

```diff
@@ -4,6 +4,8 @@ timeStamp=$(date "+%Y.%m.%d-%H.%M.%S")
 dbFolder=_mysql
 optFolder=_opt
 confFolder=_config
+cassandraDeprecationVersion="9.0.0"
```

Comment:
@shubham-kickdrumtech I'd consider adding a comment before this line, so that it will be clear this version number will need to be updated.

CC: @gneb

Reply:
@damianoferrari I have added the comment.